install/helm/gloo/generate/values.go

```diff
@@ -446,6 +446,7 @@ type GatewayProxy struct {
 	IstioMetaClusterId             *string                      `json:"istioMetaClusterId,omitempty" desc:"ISTIO_META_CLUSTER_ID Environment Variable. Defaults to \"Kubernetes\""`
 	IstioDiscoveryAddress          *string                      `json:"istioDiscoveryAddress,omitempty" desc:"discoveryAddress field of the PROXY_CONFIG environment variable. Defaults to \"istiod.istio-system.svc:15012\""`
 	EnvoyLogLevel                  *string                      `json:"envoyLogLevel,omitempty" desc:"Level at which the pod should log. Options include \"trace\", \"info\", \"debug\", \"warn\", \"error\", \"critical\" and \"off\". Default level is info"`
+	EnvoyStatsConfig               map[string]interface{}       `json:"envoyStatsConfig,omitempty" desc:"Envoy statistics configuration such as tagging. https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/metrics/v3/stats.proto#config-metrics-v3-statsconfig"`
```

Comment:
docs suggestion, if you dont mind!
```suggestion
	EnvoyStatsConfig               map[string]interface{}       `json:"envoyStatsConfig,omitempty" desc:"Envoy statistics configuration, such as tagging. For more info, see https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/metrics/v3/stats.proto#config-metrics-v3-statsconfig"`
```

Reply:
Thank you for updating the PR 🙏🏼

Reply:
Heyyo! Wondering if you were planning on committing the suggestions or not. If not, that's alright and I'll approve the PR, but if so I'll approve afterward (note: if you do commit this, i think you'll need to run code generation once more) 👀

Reply:
Oop, looks like it got 2 approvals already so you can ignore me lol. Great work/changes!

Reply:
We needed a rebase, and I did want to get that change in, and I thought I had committed it (resulting in the comment "Thank you for updating the PR"), so throwing this in.